src/gui/editors/SongEditor.cpp

```diff
@@ -728,13 +770,34 @@ SongEditorWindow::SongEditorWindow(Song* song) :
 	zoomToolBar->addWidget( zoom_lbl );
 	zoomToolBar->addWidget( m_zoomingComboBox );
 
+	DropToolBar *snapToolBar = addDropToolBarToTop(tr("Snap controls"));
+	QLabel * snap_lbl = new QLabel( m_toolBar );
+	snap_lbl->setPixmap( embed::getIconPixmap( "quantize" ) );
+
+	//Set up quantization/snapping selector
+	m_snappingComboBox = new ComboBox( m_toolBar );
+	m_snappingComboBox->setFixedSize( 80, 22 );
+	m_snappingComboBox->setModel(m_editor->m_snappingModel);
+	m_snappingComboBox->setToolTip(tr("Clip snapping"));
+
+	m_setProportionalSnapAction = new QAction(embed::getIconPixmap("proportional_snap"),
+											 tr("Toggle proportional snap on/off"), this);
+	m_setProportionalSnapAction->setCheckable(true);
+	m_setProportionalSnapAction->setChecked(true);
```

Comment:
```suggestion
	m_setProportionalSnapAction->setChecked(false);
```
Default to proportional snap being disabled.